Vagrantfile

```diff
@@ -69,7 +69,7 @@ def chef_defaults(chef, name, environment = 'splunk_server')
 end
 
 Vagrant.configure('2') do |config|
-  config.vm.box = 'bento/centos-7.7'
+  config.vm.box = 'bento/centos-8'
```

Comment:
Looks like 8.3 is the current latest version.  In order to reduce the chances of something breaking unexpectedly, I would prefer to specify that version explicitly here.  Otherwise if something in 8.4 or 8.5 were to break our test instances they would just start failing when the -8 pointer was updated with that latest release.

Reply:
updated and added conditional to add the correct repo for 8